### PR TITLE
Remove rate limit, LUA code and POW server

### PR DESCRIPTION
**Pull request by agusaldasoro**
This PR removes all the LUA code, rate limit on Nginx config and the CI check for POW server.
Also, this PR removes pow-auth-server from the `./init.sh` script so it won't be started, while `./stop` script will stop it.

## What's next?
- Remove lua imports from [decentraland's openresty docker image](https://github.com/decentraland/openresty-lua/blob/main/Dockerfile)
- Remove pow-server from docker-compose

### CODE REVIEW

File: init.sh [hunk 1]
```diff
@@ -355,7 +355,7 @@ fi
 
 echo "## Restarting containers... "
 docker-compose down
-docker-compose -f docker-compose.yml -f "platform.$(uname -s).yml" up -d
+docker-compose -f docker-compose.yml -f "platform.$(uname -s).yml" up -d nginx
```
Comment (pablitar) [inline init.sh:358]: Does this only restart nginx when we run `./init.sh`?

File: stop.sh [hunk 2]
```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 VARIABLE_SUM=$#
 CONTAINER_NAMES=$@
-CONTAINER_VALID_LIST="nginx lambdas content-server comms-server"
+CONTAINER_VALID_LIST="nginx lambdas content-server comms-server pow-auth-server"
```
Comment (pablitar) [inline stop.sh:4]: Is this OK to be added here now that it has been removed from everywhere else?

Is this to be able to delete the container even when is not on the compose file anymore?

Comment (pablitar) [reply stop.sh:4]: I get it. Pow server hasn't been removed from docker compose yet. Nevermind.

Comment (agusaldasoro) [reply stop.sh:4]: That's it. I've tested all the changes so they are able to tier down POW server and not running it anymore!

File: init.sh (hunk 1 above)
Comment (agusaldasoro) [reply init.sh:358]: `docker-compose down` will stop every running container that is defined in the docker-compose file.
`docker-compose up nginx` will only run ngnix, but as everything else is defined as a dependency, then every image (except for POW) will be run.

Comment (pablitar) [reply init.sh:358]: Awesome. Great thinking

**Review by pablitar — APPROVED** [review]
LGTM! Great work!

**Pull request merged**